magefiles/test.go

```diff
@@ -18,7 +18,7 @@ func (t Test) All() error {
 
 // Unit runs the unit tests.
 func (Test) Unit() error {
-	return RunSh("go", WithV())("test", "./...")
+	return RunSh("go", WithV())("test", "-race", "-count=1", "-coverpkg=./...", "-covermode=atomic", "-coverprofile=coverage.txt", "./...")
```

Comment:
will this make the inner loop slower?

Reply:
What inner loop?

It should make the tests slower but it shouldn't be a big difference

Reply:
With race and coverage
```
mage test:unit  17.00s user 11.78s system 202% cpu 14.205 total
```

with race
```
mage test:unit  17.76s user 18.04s system 252% cpu 14.167 total
```

with coverage
```
mage test:unit  14.46s user 15.96s system 253% cpu 12.013 total
```